content/posts/newsletter-012/index.md

```diff
@@ -660,6 +660,90 @@ about their experience of using Rust in games at [Gameroasters].
 
 ## Library & Tooling Updates
 
+### [A Servo Browser Plugin for Unity][servo-unity-post]
```

Comment:
lol, so finally Unity is going to support WebGPU by running it inside Serve browser embedded into a Unity scene 😆